src/components/dashboard/Dashboard.scss

```diff
@@ -80,62 +93,68 @@
     }
 }
 
-@include medium-screen {
-    .Dashboard {
-        position: fixed;
-        top: 6.8em;
-        left: 0;
-        right: 0;
-        bottom: 0;
-        padding: 0 2em;
-        overflow-y: scroll;
+.Dashboard {
+    position: fixed;
+    top: 6.8em;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    padding: 0 2em;
+    overflow-y: scroll;
+}
+
+.Dashboard-favorites,
+.Dashboard-shortcuts {
+    font-size: 1.6em;
+    max-width: 60em;
+}
+
+.Dashboard-favorites {
+    margin: 3em auto 1em;
+
+    @include small-screen {
+        margin-top: 1.5em;
     }
 
-    .Dashboard-favorites,
-    .Dashboard-shortcuts {
-        font-size: 1.6em;
-        max-width: 60em;
+    &::after {
+        content: "";
+        display: block;
+        clear: both;
     }
 
-    .Dashboard-favorites{
-        margin: 3em auto 1em;
+    & > li {
+        float: left;
+        width: 32%;
+        margin: 0 0 1em 2%;
 
-        &::after {
-            content: "";
-            display: block;
-            clear: both;
+        @include small-screen {
+            width: 100%;
         }
 
-        & > li {
-            float: left;
-            width: 32%;
-            margin: 0 0 1em 2%;
-
-            &:nth-child(1), &:nth-child(4), &:nth-child(7) {
-                margin-left: 0;
-                clear: both;
-            }
+        &:nth-child(1), &:nth-child(4), &:nth-child(7) {
+            margin-left: 0;
+            clear: both;
         }
-    }
-
-    .Dashboard-shortcuts {
-        margin: 0 auto;
 
-        li{
-            margin-left: 1em;
-        }
     }
+}
 
-    .Dashboard-widgets{
-        margin: 0.833em 3.2em;
-        -webkit-column-count: 2;
-        -moz-column-count: 2;
-        column-count: 2;
+.Dashboard-shortcuts {
+    margin: 0 auto;
 
+    li {
+        margin-left: 1em;
     }
+}
 
-    .Dashboard-widgets .DraggableWidget-widgetcontainer {
-        display: inline-block;
-        width: 100%;
-    }
+.Dashboard-widgets {
+    margin: 0.833em 3.2em;
+    -webkit-column-count: 2;
+    -moz-column-count: 2;
```

Comment:
Perhaps we can drop the prefixes, now?

https://caniuse.com/#feat=multicolumn

(According to the above link, Jan 2017 saw the last release of a Firefox which needed `-moz-` prefix on column-count.)

Reply:
(Totally a separate issue!)